src/containers/LoginModal/LoginModal.tsx

```diff
@@ -78,12 +78,12 @@ export default class LoginModal extends React.PureComponent<Props, State> {
   }
 
   render() {
-    const { open, className, isLoading, onClose } = this.props
+    const { className, isLoading, onClose } = this.props
     const { hasError } = this.state
 
     return (
       <BaseLoginModal
```

Comment:
It's necessary to use the Modal that's operated using the store in order to open it through the store.

Reply:
The `BaseLoginModal` comes from the UI package, the one operated using the store is this `LoginModal`.